client/src/fabric/tensegrity-brick.ts

```diff
@@ -90,6 +91,7 @@ export function createBrickOnOrigin(fabric: TensegrityFabric, altitude: number):
 }
 
 export function createBrickOnFace(face: IFace): IBrick {
+    const negativeFace = TRIANGLE_ARRAY[face.triangle].negative
```

Comment:
Has to be a better name for `TRIANGLE_ARRAY`

Reply:
TRIANGLE_DEFINITIONS